src/js_realm.hpp

```diff
@@ -598,6 +600,9 @@ inline typename T::Function RealmClass<T>::create_constructor(ContextType ctx)
         realm::disable_sync_to_disk();
     }
 
+    std::shared_ptr<util::NullLogger> logger = std::make_shared<util::NullLogger>();
+    util::Logger::set_default_logger(std::move(logger));
```

Comment:
Wouln't it make more sense to set the default log level?

Reply:
`NullLogger` is setting it to `off` so no need

Reply:
But with this, users can't set the log level to something else and get an output?

Reply:
Fair.

Reply:
See my comment below